example/errormsg/errormsg_test.go

```diff
@@ -37,100 +40,77 @@ func parse(t *testing.T, code string, expectPass bool) {
 	}
 }
 
-func TestParsedErrors(t *testing.T) {
-	t.Run("pass", func(t *testing.T) {
-		candidates := []string{
-			"var abcd = 123;",
-			"  var  _  :=  abcd  ;",
-			"var a  = 1.23 ;",
-			"var x;",
-		}
-		for _, candidate := range candidates {
-			candidate := candidate
-			t.Run(candidate, func(t *testing.T) {
-				parse(t, candidate, true)
-			})
-		}
-	})
-
-	t.Run("EOF", func(t *testing.T) {
-		parse(t, "var a = 1", false) // missing ';'
```

Comment:
Did we loose these tests?

Reply:
Nope, I split the function into two separate tests, both of which use a loop to test an incremental set of conditions:

https://github.com/goccmack/gocc/pull/115/files/b9ce84a3a28692219f9e74f8e8e6303f17aa1af0#diff-75dcf2f68640e310c0b364b48d66257a16caabe6d53291b22367f3198c59212dR43-R75